### ISSUE TITLE
Implement ramps

### ISSUE THREAD

**Issue by jjppof** [jjppof/goldensun_html5#246]
![ramp](https://user-images.githubusercontent.com/7890647/107134266-4203db00-68cf-11eb-9793-39b635ef7030.gif)

**Comment by PaulB-H** [comment]
Noticed in the demo the ramp works, is this already done?

**Comment by jjppof** [comment]
> Noticed in the demo the ramp works, is this already done?

Well, at least this kind of ramp we still don't have in the demo. Where did you see it?

**Comment by PaulB-H** [comment]
![screengrab_ledge](https://user-images.githubusercontent.com/51964537/115558257-8bce5180-a280-11eb-9f40-3be6448652c1.gif)

**Comment by jjppof** [comment]
> Does this one work differently?
> 
> ![screengrab_ledge](https://user-images.githubusercontent.com/51964537/115558257-8bce5180-a280-11eb-9f40-3be6448652c1.gif)

Yes. Besides being a lateral movement, in the original gif, the hero translates on the y-axis even if you're pressing only right or left. In your gif, it's just a speed reduction. Maybe the SpeedEvent needs some new properties to deal with this translation.

Btw, are you using Firefox?

**Comment by PaulB-H** [comment]
Ah okay thanks, I have a lot to figure out still 😅

Yeah I was for that, Chrome really is worlds smoother though wow
